lib/util/constants.js

```diff
@@ -0,0 +1,19 @@
+const GHAE_URL_PATTERN_PROD = 'ghe.com';
+const GHAE_URL_PATTERN_DEV = 'ghaekube.net';
+const GHAE_INSTANCE_URL_DEV = 'https://ghaebuild48746.ghaekube.net';
```

Comment:
change 'https://ghaebuild48746.ghaekube.net' to "https://GhitHubAEAccount.ghaekube.net"